site/docs/v1/tech/identity-providers/external-jwt/index.adoc

```diff
@@ -120,9 +127,10 @@ For example, if you select `company` as the incoming claim, and specify `Registr
 
 == Managed Domains
 
-In order to successfully reconcile a JWT to FusionAuth using this identify provider, you will need to specify one more managed domains. A managed domain will indicate to FusionAuth that this identify provider is able to reconcile a user based upon their email address.
+To successfully reconcile a JWT to FusionAuth using this identity provider, you can *optionally* specify one more managed domains.
+A managed domain will indicate to FusionAuth this identity provider can reconcile a user based upon their email address.
 
-For example if a user's email address is `jared@piedpiper.com`, in order for a JWT owned by Jared to be reconciled to FusionAuth using this identity provider, `piedpiper.com` will need to be configured as a managed domain.
+For example, if a user's email address is `jared@piedpiper.com`, in order for a JWT owned by Jared to be reconciled to FusionAuth using this identity provider, `piedpiper.com` could be configured as a managed domain.
```

Comment:
Can you add to the end: "You may leave this blank. If you do so, then any email can be used with this identity provider. This is helpful if you are not certain which email domains your users will have."